configure.ac

```diff
@@ -6420,6 +6420,7 @@ plugin_tcpconns="no"
 plugin_ted="no"
 plugin_thermal="no"
 plugin_turbostat="no"
+plugin_ubi="no"
```

Comment:
You also need to add `plugin_ubi="yes"` few lines below, enabling it for Linux by default, like others.

Reply:
Not sure if we want to enable by default, because it only makes sense for the NAND device using the UBIFS filesystem. What do you think ?

Reply:
We do not enable it by default, but _compile_ by default )

Reply:
Ok, thanks!